src/ddmd/cppmangle.d

```diff
@@ -746,61 +787,50 @@ public:
         {
             assert(t.basetype && t.basetype.ty == Tsarray);
             assert((cast(TypeSArray)t.basetype).dim);
-            //buf.printf("Dv%llu_", ((TypeSArray *)t.basetype).dim.toInteger());// -- Gnu ABI v.4
-            buf.writestring("U8__vector"); //-- Gnu ABI v.3
+            version (all)
+                buf.printf("Dv%llu_", (cast(TypeSArray *)t.basetype).dim.toInteger()); // -- Gnu ABI v.4
+            else
+                buf.writestring("U8__vector"); //-- Gnu ABI v.3
```

Comment:
Change in vector abi could be a separate commit.

Reply:
Yeah, I'll do that separately. https://issues.dlang.org/show_bug.cgi?id=17948